events/events-team/comms/README.md

```diff
@@ -1,55 +1,191 @@
+- Claim and write the announcement comms (email &/or social, slack, etc.)
+- Share Plan Docs with team
+  - Ensure shadows and volunteers are signed up for necessary activities
+- Ensure comms leading up to the event are planned, prioritized, and completed successfully
+- [Post-KCS] Update role handbook
+- [Post-KCS] Update KCS Comms Overall Plan Template doc
+
+## Shadow to the KCS Comms Lead Role
+
+Shadows are expected to serve as KCS Comms Leads in a future cycle. During their shadowing cycle(s), Shadows will work closely with the KCS Comms Lead to learn both how to run the KCS Comms team and how to create comms themselves. By signing up to shadow, you are committing to be involved with the event team for 1.5 years or 3 comms cycles (shadow, lead, and emeritus). A Shadow candidate must either have [Kubernetes Org Membership] or be planning to pursue org membership.
+
+[Kubernetes Org Membership] can be obtained through KCS Comms work, and a shadow should complete enough tasks to be eligible for org membership over the course of a cycle. Do note that org membership is an indication of active and continuous contribution to the Kubernetes project/community. Shadows are expected to be committed to contributing to Kubernetes on a long-term basis.
+
+### Requirements
+* Review this document.
+* A KCS Comms Shadow must either have, or plan to pursue, [Kubernetes Org Membership].
+* Commit to contributing to the KCS Staff team for multiple cycles.
+
+## Tasks
+During the course of their KCS cycle as a shadow, KCS Comms Shadows are expected to complete tasks including but not limited to:
+* Joining the appropriate Slack channels for KCS staff collaboration and for sending comms about KCS to the community
+* Working with the Comms Lead and Contributor Comms Subproject to get access to tooling for posting on Kubernetes Contributor-focused social media channels.
+* Representing the KCS Comms team in KCS Staff meetings.
+* Writing email comms as instructed by the KCS Comms Lead.
+* Writing social media comms as instructed by the KCS Comms Lead.
+* Reviewing comms written by other shadows, volunteers, or the lead as directed by the KCS Comms Lead
+
+Shadows should work with the Comms Lead to ensure they are getting sufficient experience to feel comfortable taking on the KCS Comms Lead role in a future cycle after completion of a shadow cycle.
+
+In the event a shadow is unable to complete the tasks expected of them as designated by the KCS Comms Lead, they should inform the KCS Comms Lead as early as possible, to the best of their ability. Shadows who are unable to complete the required tasks as designated by the KCS Comms Lead, will be removed from the list of shadows. Upon failure to complete a shadowing cycle, the contributor may request to become a shadow in a future cycle. Successful completion of a shadowing cycle (as determined by the KCS Comms Lead) or equivalent experience is required to become a KCS Comms Lead.
```

Comment:
I would actually bold the, "as early as possible." And maybe throw in some language that life happens and we support you in your endeavours be they here or elsewhere.